internal/app/machined/pkg/controllers/network/operator/dhcp6.go

```diff
@@ -149,7 +149,7 @@ func (d *DHCP6) parseReply(reply *dhcpv6.Message) {
 	d.mu.Lock()
 	defer d.mu.Unlock()
 
-	if reply.Options.OneIANA() != nil {
+	if reply.Options.OneIANA() != nil && reply.Options.OneIANA().Options.OneAddress() != nil {
```

Comment:
would any of these intermediate methods return nil?

Reply:
nope, they are just plain fields without pointers. I have only panic message, some my guess is that `OneAddress()` is nil, as it might return nil according to the code